Directory.Packages.props

```diff
@@ -86,78 +86,78 @@
+    <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.9.0-beta.1" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.9.0" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.9.0" />
     <!-- build dependencies -->
     <PackageVersion Include="MicroBuild.Plugins.SwixBuild.Dotnet" Version="1.1.87-gba258badda" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.9.2" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.10.0" />
     <PackageVersion Include="Microsoft.DotNet.Build.Tasks.Workloads" Version="8.0.0-beta.23564.4" />
     <PackageVersion Include="Microsoft.Signed.Wix" Version="$(MicrosoftSignedWixVersion)" />
     <PackageVersion Include="Microsoft.DotNet.Build.Tasks.Installers" Version="8.0.0-beta.23564.4" />
     <PackageVersion Include="System.CommandLine" Version="2.0.0-beta4.24102.1" />
     <!-- unit test dependencies -->
-    <PackageVersion Include="bUnit" Version="1.27.17" />
-    <PackageVersion Include="JsonSchema.Net" Version="6.0.4" />
+    <PackageVersion Include="bUnit" Version="1.28.9" />
+    <PackageVersion Include="JsonSchema.Net" Version="7.1.0" />
     <PackageVersion Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotNetRemoteExecutorPackageVersion)" />
     <PackageVersion Include="Microsoft.DotNet.XUnitExtensions" Version="$(MicrosoftDotNetXUnitExtensionsPackageVersion)" />
-    <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="$(MicrosoftExtensionsDiagnosticsTestingPackageVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="8.6.0" />
     <PackageVersion Include="Microsoft.NET.Runtime.WorkloadTesting.Internal" Version="$(MicrosoftNETRuntimeWorkloadTestingInternalVersion)" />
-    <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.5.0" />
+    <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.6.2" />
     <PackageVersion Include="Testcontainers.MongoDb" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.MsSql" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.MySql" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.PostgreSQL" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.RabbitMq" Version="$(TestcontainersPackageVersion)" />
+    <PackageVersion Include="Testcontainers.MsSql" Version="3.9.0" />
+    <PackageVersion Include="Testcontainers.MySql" Version="3.9.0" />
+    <PackageVersion Include="Testcontainers.PostgreSQL" Version="3.9.0" />
+    <PackageVersion Include="Testcontainers.RabbitMq" Version="3.9.0" />
     <PackageVersion Include="Testcontainers.Redis" Version="$(TestcontainersPackageVersion)" />
```

Comment:
1. Why are removing the common MSBuild property in the other lines?
2. Why isn't this one updated like the rest of them?

Note that TestContainers are only used by our tests, so updating it here isn't buying any customer value.

Reply:
I noticed msbuild properties were removed ( I used a tool for the updates) I will fix that.

Reply:
>Note that TestContainers are only used by our tests, so updating it here isn't buying any customer value.

Good call, reverting that.